src/routes/onboarding.tsx

```
@@ -37,175 +29,36 @@ export const Route = createFileRoute('/onboarding')({
+      // Redirect to admin
       navigate({
         to: '/$username/admin/editor/profiles',
-        params: { username: profile.username },
+        params: { username: newUsername },
       })
     } catch (error) {
-      setFormError(error instanceof Error ? error.message : 'Something went wrong.')
-    } finally {
-      setIsSubmitting(false)
+      throw error
     }
```

ESLint

[error] Unnecessary try/catch wrapper.

(no-useless-catch)